components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -91,6 +91,20 @@ type Config struct {
 	APIToken        string         `json:"api_token"`
 }
 
+func (c *Config) NewConfig() *Config {
+	return &Config{
+		Hardware:    &Hardware{},
+		Certificate: []*Certificate{},
+		SSHUser:     &SSHUser{},
+		Backup: &Backup{
+			FileSystem:    &FileSystem{},
+			ObjectStorage: &ObjectStorage{},
+		},
+		ExternalOS: &ExternalOS{},
```

Comment:
We can remove this and set them only if needed. That way it ensure that either the field has value or it is nil